concepticondata/conceptlists/Hantgan-2022-348.tsv

```diff
@@ -0,0 +1,349 @@
+Hantgan-2022-348-174	174	LOCK	1596	LOCK	couvercle; fermeture.
+Hantgan-2022-348-175	175	LOOK	1819	LOOK	examiner, regarder
+Hantgan-2022-348-176	176	LOOK FOR	1468	LOOK FOR	chercher
+Hantgan-2022-348-177	177	LOUSE	1392	LOUSE	pou
+Hantgan-2022-348-178	178	LUNCH	768	LUNCH	déjeuner
+Hantgan-2022-348-179	179	LUNG	688	LUNG	poumon
+Hantgan-2022-348-180	180	MAIZE	506	MAIZE	maïs
+Hantgan-2022-348-181	181	MAN	1554	MAN	homme
+Hantgan-2022-348-182	182	MARKET	633	MARKET	marché
+Hantgan-2022-348-183	183	MARROW	1835	MARROW	moelle
+Hantgan-2022-348-184	184	MARRY	203	MARRY	(homme) épouser (femme)
+Hantgan-2022-348-185	185	MAT	195	MAT	natte
+Hantgan-2022-348-186	186	MEAL	1526	MEAL	repas
+Hantgan-2022-348-187	187	MEAT	634	MEAT	viande
+Hantgan-2022-348-188	188	MEDICINE	1372	MEDICINE	médicament
+Hantgan-2022-348-189	189	MEET	1831	MEET	(individus) se rassembler, se réunir
+Hantgan-2022-348-190	190	MILK	635	MILK	lait
+Hantgan-2022-348-191	191	MILLET	931	MILLET	mil
+Hantgan-2022-348-192	192	MONEY	1559	MONEY	argent
+Hantgan-2022-348-193	193	MOON	1313	MOON	lune
+Hantgan-2022-348-194	194	MOSQUE	1603	MOSQUE	mosquéé
+Hantgan-2022-348-195	195	MOTHER	1216	MOTHER	mère
+Hantgan-2022-348-196	196	MOUSE	1491	MOUSE	souris
+Hantgan-2022-348-197	197	MOUTH	674	MOUTH	bouche
+Hantgan-2022-348-198	198	MUD	640	MUD	boue
+Hantgan-2022-348-199	199	NAME	1405	NAME	nom
+Hantgan-2022-348-200	200	NARROW	1267	NARROW	coincé
+Hantgan-2022-348-201	201	NEAR	1942	NEAR	proche
+Hantgan-2022-348-202	202	NECK	1333	NECK	cou
+Hantgan-2022-348-203	203	NEED (NOUN)	643	NEED (NOUN)	besoin
+Hantgan-2022-348-204	204	NEIGHBOUR	1332	NEIGHBOUR	voisin
+Hantgan-2022-348-205	205	NEW	1231	NEW	nouveau
+Hantgan-2022-348-206	206	NIGHT	1233	NIGHT	nuit
+Hantgan-2022-348-207	207	NINE	1483	NINE	neuf
+Hantgan-2022-348-208	208	NOBLE	482	NOBLE	noble (non esclave)
+Hantgan-2022-348-209	209	NOSE	1221	NOSE	nez
+Hantgan-2022-348-210	210	NOW	1376	NOW	maintenant
+Hantgan-2022-348-211	211	NURSE	1104	NURSE	(femelle) allaiter, faire téter (enfant)
+Hantgan-2022-348-212	212	OLD	1229	OLD	âgé, vieux (person)
+Hantgan-2022-348-213	213	OLD MAN	406	OLD MAN	vieil homme
+Hantgan-2022-348-214	214	OLD WOMAN	907	OLD WOMAN	vieille femme
+Hantgan-2022-348-215	215	ONE	1493	ONE	un
+Hantgan-2022-348-216	216	ONION	2366	ONION	Allium (diverses var. cultivéés), oignon
+Hantgan-2022-348-217	217	OPEN	1156	OPEN	découvrir (qch, qn), ouvrir (porte, récipient)
+Hantgan-2022-348-218	218	ORPHAN	1149	ORPHAN	orphelin
+Hantgan-2022-348-219	219	OTHER	197	OTHER	autre
+Hantgan-2022-348-220	220	PATH OR ROAD	2457	PATH OR ROAD	chemin
+Hantgan-2022-348-221	221	PENIS	1222	PENIS	membre viril, sexe de l’homme [common in insults; TSK béndè] [SYN:zòfól-ò ] -poss (inal): 1Sg bènd-èy, 3Sg bènd-ò , 3FullSg .ángá bènd-ô -ex of insult: bà:b-à.­bénd-ò your father’s prick!
+Hantgan-2022-348-222	222	PERSON	683	PERSON	chose ou personne jolie [DIMIN in form].
+Hantgan-2022-348-223	223	PESTLE	225	PESTLE	pilon
+Hantgan-2022-348-224	224	PICK	7	GATHER	cueiller (fruit)
```

Comment:
Due to the fruit in parentheses, would 2148 PICK be a better mapping?

Reply:
Yes, in my opinion.

Reply:
Should I check over everything now?